civicrm.php

```diff
@@ -1419,52 +1419,76 @@ private function restore_wp_magic_quotes() {
    *
    * @since 4.4
    *
-   * @return boolean True if request is for a CiviCRM page, false otherwise.
+   * @return boolean $is_page True if request is for a CiviCRM page, false otherwise.
    */
   public function is_page_request() {
 
     // Assume not a CiviCRM page.
-    $return = FALSE;
+    $is_page = FALSE;
 
-    // Kick out if not CiviCRM.
+    // Bail if no CiviCRM.
     if (!$this->initialize()) {
       return $return;
     }
 
-    // Get args.
+    // Get request args.
     $argdata = $this->get_request_args();
 
-    // Grab query var.
+    // Try and populate "html" query var for testing snippet requests.
     $html = get_query_var('html');
     if (empty($html)) {
       // We do not use $html apart to test for empty.
-      // phpcs:disable WordPress.Security.NonceVerification.Recommended
-      // phpcs:disable WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
+      // phpcs:ignore WordPress.Security.NonceVerification.Recommended, WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
       $html = isset($_GET['html']) ? wp_unslash($_GET['html']) : '';
-      // phpcs:enable WordPress.Security.NonceVerification.Recommended
-      // phpcs:enable WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
     }
 
     /*
      * FIXME: It's not sustainable to hardcode a whitelist of all of non-HTML
      * pages. Maybe the menu-XML should include some metadata to make this
```

Comment:
Oh look, there's the same comment I made in the other PR, probably from our conversation about this 10 years ago 👴🏻